react-app/src/reducers/index.js

```diff
@@ -1,13 +1,23 @@
 import { combineReducers } from 'redux';
 
+import { SIGN_OUT } from 'actions/account';
+
 import account from './account';
 import entries from './entries';
 import entriesByDate from './entriesByDate';
 
-const rootReducer = combineReducers({
+const appReducer = combineReducers({
   account,
   entries,
   entriesByDate,
 });
 
+const rootReducer = (state, action) => {
+  if (action.type === SIGN_OUT) {
```

Comment:
I don't think this is pretty common or really obvious why you do this.
I'd like to have a comment somewhere near this to explain why we need it.

Reply:
👍

Reply:
great comment.